components/markets/ScalarPriceRange.tsx

```diff
@@ -28,44 +29,20 @@ const ScalarPriceRange = ({
   const inferedType: ScalarRangeType = scalarType ?? "number";
```

Comment:
Do you need to do a null check here since its also done and inferred in the formatting function?